.env.testnet

```diff
@@ -14,6 +14,7 @@ REACT_APP_TRANSAK_API_KEY=a2d23229-58f9-4ca5-a3d4-2c3d2b1b9a81
 REACT_APP_TRANSAK_ENV=STAGING
 
 REACT_APP_GOOGLE_ANALYTICS=GTM-KTSX2PL
+REACT_APP_GOOGLE_ANALYTICS_CLICKTHROUGH=GTM-5MWCJN3
```

Comment:
Do we want it on testnet as well (I mean both of the GTMs)?

Reply:
removed both of the analytics ids from this file as discussed on Jira